.github/bin/generate-coverage-html.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2021 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Generate a directory with html coverage. This prepares it to be uploaded
+# to a location to serve or to inspect locally.
+#
+# NOTE: The CI uses codecov, but this script is useful locally.
+#   In the actions, we upload the coverage to codecov.io currently, so
+#   this HTML generation script is not used in the CI, but is still
+#   useful for local coverage checking before sending a pull request.
+#   In case we stop using codecov, this is the starting point to generate
+#   and upload coverage.
+#
+# This requires the `gcov` package to be installed.
+#
+# Precondition, having run bazel coverage with coverage data collection, e.g.
+#   MODE=coverage .github/bin/build-and-test.sh
+#
+
+PROJECT_ROOT=$(dirname $0)/../../
+OUTPUT_DIR=coverage-html
+
+cd $PROJECT_ROOT
+genhtml -o ${OUTPUT_DIR} bazel-out/_coverage/_coverage_report.dat
```

Comment:
genhtml is a tool from another required package?  if so, include in comment above.

Reply:
It is mentioned above in the readme: `gcov` is the package to install.
That is also the package to install to allow bazel coverage to work in the first place.

Reply:
mmh, maybe it is lcov, let me check.

Reply:
It was `lcov`. Fixed.